### ISSUE TITLE
Input and button are not announced when used with VoiceOver + Chrome

### ISSUE THREAD

**Issue by dimazawr** [duetds/date-picker#99]
**Describe the bug**
When date-picker is used together with VoiceOver and Chrome on macOS input and button label get announced just once, but not when the date is selected. 

**To Reproduce**
1. Switch on VoiceOver and open Chrome
2. Navigate to date-picker with keyboard
3. Open date-picker via button
4. Select date
5. Listen that VoiceOver announces "Selected, menu 9 items, .."
6. Navigate with keyboard to any other element on the page
7. Navigate back to date-picker input or button
8. Confirm that neither labels or value are getting announced. 

**Expected behavior**
Would be nice to hear the labels and selected value, what if user navigates back and forth after choosing the date for the first time?

**Desktop (please complete the following information):**
 - OS: macOs 11.6
 - Browser: Chrome
 - Version: 97.0.4692.71

**Additional context**
Works in Firefox and Safari
